core/math/random_number_generator.h

```diff
@@ -57,6 +57,20 @@ class RandomNumberGenerator : public Reference {
 
 	_FORCE_INLINE_ real_t randfn(real_t mean = 0.0, real_t deviation = 1.0) { return randbase.randfn(mean, deviation); }
 
+	_FORCE_INLINE_ Vector2 randv_circle(real_t p_radius = 1.0) {
+		real_t r = Math::sqrt(randbase.random(0.0, 1.0)) * p_radius;
+		real_t t = randbase.random(0.0, 1.0) * Math_TAU;
+		return Vector2(r * Math::cos(t), r * Math::sin(t));
+	}
+
+	_FORCE_INLINE_ Vector2 randv_range_circle(real_t p_min_radius = 0.0, real_t p_max_radius = 1.0) {
```

Comment:
Subjectively, I think `randv_range_circle` should be named `randv_circle_range`(to filter it alphabetically) or even `randv_ring` (radically)

Reply:
I like `randv_ring`, and perhaps `range` is not the best choice indeed here because it might falsely suggest that it's going to generate *arc* range rather than *radius* range.

Reply:
Renamed `randv_range_circle` → `randv_ring` for now. 🙂